python/README.md

```diff
@@ -29,4 +29,4 @@ The Python packaging for Spark is not intended to replace all of the other use c
 
 ## Python Requirements
 
-At its core PySpark depends on Py4J (currently version 0.10.6), but additional sub-packages have their own requirements (including numpy and pandas).
+At its core PySpark depends on Py4J (currently version 0.10.6), but additional sub-packages have their own requirements (including numpy, pandas, and pyarrow).
```

Comment:
This sounds like mandatory, but I think pyarrow is still an optional choice. Right?

Reply:
Yea, Pandas and PyArrow are optional. Maybe, it's nicer if we have some more details here too.

Reply:
I added some more details. WDYT?